browser/about_flags.cc

```diff
@@ -146,6 +147,10 @@ constexpr char kBraveExtensionNetworkBlockingName[] =
 constexpr char kBraveExtensionNetworkBlockingDescription[] =
     "Enable blocking for network requests initiated by extensions";
 
+constexpr char kBraveReduceLanguageName[] = "Reduce language identifiability";
```

Comment:
in all cases I think we should be clear that this is about accept-language

Reply:
well except it's more than that because the same setting controls font whitelisting

Reply:
Yea, seconding @pilgrim-brave , this setting controls both the explicit language preferneces in accept-language and navigator.languages, but also the implicit language preferences based on fonts available on the system

Reply:
I still think `kBraveReduceLanguage` is going to be basically meaningless to anyone looking at the code. Other suggestions?